Casks/d/dorico.rb

```diff
@@ -1,15 +1,15 @@
 cask "dorico" do
-  version "5.1.81,97b84063-5556-4ef4-b308-15edc4d72339"
-  sha256 "c3cf427d3ddf8e7e9e9f751c7896fe869b14c97cc9a2736ac030dce5be19fa91"
+  version "6,c2180b3c-bfcb-4cd1-afc5-ea41e618c728"
+  sha256 "f99ba5ea6d5ee69fde8f4707c62ea6162198fe79eb2ff14dd72f2ddc24a4a603"
 
   url "https://download.steinberg.net/automated_updates/sda_downloads/#{version.csv.second}/Dorico_#{version.csv.first}_Installer_mac.dmg"
   name "Dorico"
   desc "Scoring software"
   homepage "https://www.steinberg.net/dorico/"
 
   livecheck do
-    url "https://o.steinberg.net/en/support/downloads/dorico_#{version.csv.first.major}/dorico_pro_#{version.csv.first.major}.html"
-    regex(%r{href=.*?downloads/([^/]+)/Dorico[._-]v?(\d+(?:\.\d+)+)[._-]Installer[._-]mac\.dmg}i)
+    url "https://o.steinberg.net/en/support/downloads/dorico_#{version.csv.first}.html"
```

Comment:
```suggestion
    url "https://o.steinberg.net/en/support/downloads/dorico_#{version.csv.first.major}.html"
```

We should keep this as-is, assuming the URL will use the major version. Otherwise, the `livecheck` will fail once the cask has been bumped to a minor version, e.g., `6.1`.